Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.4.0
-FROM rust:1.66-slim-bullseye as builder
+FROM --platform=linux/amd64 rust:1.66-slim-bullseye as builder
```

Comment:
does this build all future images as amd64, including on prod? or is this conditionally built somehow